agent/config/config_windows.go

```diff
@@ -94,6 +95,8 @@ func (cfg *Config) platformOverrides() {
 
 	// ensure TaskResourceLimit is disabled
 	cfg.TaskCPUMemLimit = ExplicitlyDisabled
+
+	cfg.CPUUnboundedWindowsEnabled = utils.ParseBool(os.Getenv("ECS_ENABLE_CPU_UNBOUNDED_WINDOWS_WORKAROUND"), false)
```

Comment:
do we really need the string `WORKAROUND` in the name here? Seems unnecessary.

Reply:
suggestion by @samuelkarp to name it this way was to convey this could be a temporary fix/hack, until cpu shares are fixed in windows. Moreover allowing both cpu unbounded and bounded tasks in an instance can have serious consequences like bounded tasks not getting their allotted shares and 'workaround' may warn the customers to use it with caution. wdyt?

Reply:
Just having a flag named `ECS_ENABLE_CPU_UNBOUNDED_WINDOWS` would probably indicate that this is a workaround imo. I don't have strong opinions here. The env var name does seem too long, however.

Reply:
I'd like the env var name to convey that we might remove it as a configuration option at some point (i.e., when we fix the CPU bursting behavior).  I think `WORKAROUND` or `HACK` is helpful in pointing out that this is not an intended long-term approach.